dashboard/test/models/external_link_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class ExternalLinkTest < ActiveSupport::TestCase
+  test 'prepend urls' do
+    external_link_level = create(:external_link, url: 'www.cnn.com')
+    assert 'http://www.cnn.com', external_link_level.url
```

Comment:
`assert_equal`?  Right now this test only asserts that strings are truthy in Ruby 😄 .